src/cascadia/ut_app/JsonUtilsTests.cpp

```diff
@@ -295,17 +295,30 @@ namespace TerminalAppUnitTests
     template<typename TExpected, typename TJson>
     static void TryBasicType(TExpected&& expected, TJson&& json)
     {
+        // test FromJson
         Json::Value jsonObject{ json };
         const auto value{ GetValue<TExpected>(jsonObject) };
         VERIFY_ARE_EQUAL(expected, value, NoThrowString{}.Format(L"(type: %hs)", typeid(TExpected).name()));
+
+        // test ToJson
+        {
+            const std::string key{ "myKey" };
+
+            Json::Value expectedJson{};
+            expectedJson[key] = json;
+
+            jsonObject = {};
+            SetValueForKey<TExpected>(jsonObject, key, expected);

```

Comment:
Is this template typename required? If so, there's been a type deduction fail

Reply:
Nah. Just did it to match with the `GetValue<TExpected>()` call above. removing angle brackets.